.gitpod.yml

```diff
@@ -0,0 +1,11 @@
+# Commands to start on workspace startup
+tasks:
+  - name: Dev Server
+    init: npm install --lts
```

Comment:
Oh, sorry, I wasn't careful when I commented on this line before. The `--lts` is needed for `nvm install` **not** `npm install`.

So this line should read: `init: npm install`

Reply:
If somehow we can specify Node constraints here, then we want the latest LTS version.

Reply:
thanks for the clarification! I'll look into it, we may be able to have both run.

Reply:
I've updated the .gitpod.yml, it looks like we can run  multiple commands using `|`